src/dali_backend.cc

```diff
@@ -21,7 +21,9 @@
 // SOFTWARE.
 
 #include "src/dali_backend.h"
+
 #include <memory>
+
```

Comment:
But we don't put this blank line. We just let the formatter order all the includes together.